client/e2e/protractor.conf.js

```diff
@@ -27,6 +27,6 @@ exports.config = {
     require('ts-node').register({
       project: require('path').join(__dirname, './tsconfig.json')
     });
-    jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
+    jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: 'pretty' } }));
```

Comment:
This file looks good to me. I changed this, because I updated the jasmine reporter dependency and this was a change in that code.

Reply:
Ok.